src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -334,13 +334,19 @@ export const useDenyReimbursementRequest = (id: string) => {
  *
  * @param fileIds The google file ids to fetch the images for
  */
-export const useDownloadPDFOfImages = (startDate: Date, endDate: Date) => {
+export const useDownloadPDFOfImages = (startDate: Date, endDate: Date, refundSource: String) => {
   return useMutation(['reimbursement-requests'], async (formData: { fileIds: string[] }) => {
     const promises = formData.fileIds.map((fileId) => {
       return downloadGoogleImage(fileId);
     });
     const blobs = await Promise.all(promises);
-    await downloadBlobsToPdf(blobs, `receipts-${startDate.toLocaleDateString()}-${endDate.toLocaleDateString()}.pdf`);
+    let pdfName = `receipts-${refundSource}-${startDate.toLocaleDateString()}-${endDate.toLocaleDateString()}.pdf`;
```

Comment:
you can make the last section `-${startDate.toLocaleDateString()}-...` a const and call it in both places to save space